### DISCUSSION TITLE
Visualize DeepMimic motion file

### DISCUSSION THREAD

**Discussion by tfederico** [bulletphysics/bullet3#3726] [General]
Hello,

is there any way to visualize a motion file for DeepMimic using pybullet?

**Comment by tfederico** [comment]
@erwincoumans does pybullet include such feature at the moment?

**Comment by erwincoumans** [comment]
If you run the testrl.py, the transparent figure is the DeepMimic reference motion.

**Comment by tfederico** [comment]
I was wondering if it is possible to see just the trasparent figure, without the trained character, as a "debugging" for motion files
